File: eval_corpus/projects/g_clean_twins/comments_tricky.py
"""음성 대조군: 취약해 보이는 코드가 주석/독스트링에만 존재.

과거 코드 리뷰 메모:
    eval(user_input) 같은 호출은 금지한다.
    cursor.execute(f"SELECT * FROM t WHERE x = '{x}'") 패턴도 금지.
"""


def sanitize(value: str) -> str:
    # 예전 버전은 os.system("del " + value) 를 호출했지만 제거했다.
    # exec(llm_code) 도 사용하지 않는다.
    return value.strip()

File: eval_corpus/projects/g_clean_twins/comments_tricky.js
// 음성 대조군: 취약해 보이는 코드가 주석에만 존재 (JS 주석 처리 갭 측정용)
// 과거 코드 리뷰 메모: eval(req.query.code) 는 금지한다.
// connection.query("SELECT * FROM t WHERE id = " + id) 패턴도 금지.
/*
  옛 구현:
  const fn = new Function(body);
  element.innerHTML = userInput;
*/

function sanitize(value) {
  return String(value).trim();
}

module.exports = { sanitize };
